core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -659,7 +659,7 @@ public void testCompleteCreateTable() {
   }
 
   @Test
-  public void testDefaultTableProperties() {
+  public void testDefaultTablePropertiesCreate() {
```

Comment:
no need to rename this
```suggestion
  public void testDefaultTableProperties() {
```